README.md

```diff
@@ -346,6 +346,13 @@ Type: `boolean`
 If `true`, the modal dialog's [focus trap](https://github.com/davidtheclark/focus-trap) will be paused.
 You'll want to use this prop if you have another nested focus trap *inside* the modal.
 
+### focusTrapOptions
+
+Type: `object`
+
+Customize properties of the `focusTrapOptions` prop that is passed to the modal dialog's [focus trap](https://github.com/davidtheclark/focus-trap).
+Use this prop if you need better control of where focus is returned.
```

Comment:
Let's prepend `For example, you can use this prop ...`, since this is only one of multiple use cases for the prop.